Dockerfile

```diff
@@ -14,7 +14,9 @@ RUN DEBIAN_FRONTEND=noninteractive apt-get -y update  \
       gdb                                             \
       libncurses5-dev                                 \
       apt-transport-https                             \
-      ca-certificates
+      ca-certificates                                 \
+      libstdc++-10-dev
```

Comment:
To get `concepts` into the `opt` pass plugin. The pass plugin is compiled with the system `stdlib` instead of the instrumented one we provide. It makes sense, since we don't really need to link the LLVM pass plugin against instrumented code.